pkg/cmd/grafana-cli/commands/commands.go

```diff
@@ -40,10 +42,10 @@ func runDbCommand(command func(commandLine CommandLine) error) func(context *cli
 	}
 }
 
-func runPluginCommand(command func(commandLine CommandLine) error) func(context *cli.Context) {
+func runPluginCommand(command func(commandLine utils.CommandLine) error) func(context *cli.Context) {
 	return func(context *cli.Context) {
 
-		cmd := &contextCommandLine{context}
+		cmd := &utils.ContextCommandLine{context}
```

Comment:
This started failing for:
`pkg/cmd/grafana-cli/commands/commands.go:19:11: github.com/grafana/grafana/pkg/cmd/grafana-cli/utils.ContextCommandLine composite literal uses unkeyed fields`

But not sure why as I only moved the struct to different package.